framework/base/Application.php

```diff
@@ -411,8 +414,6 @@ public function run()
      */
     abstract public function handleRequest($request);
 
-    private $_runtimePath;
-
```

Comment:
for private cache variables I prefer to keep them near the methods that use them. We are following this rule quite consistently.